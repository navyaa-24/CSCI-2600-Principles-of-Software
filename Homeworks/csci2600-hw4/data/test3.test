r
7
11
r
81
20
r
Lally
Lally Hall
r
Troy Building
11
r
Troy Building
87 Gymnasium
r
bad1
bad2
r
Lally Hall
bad2
r
bad1
bad1
r
20
81
r
20
21
r
53
22
r
23
53
r
108
108
r
Lally Hall
108
r
108
Lally Hall
r
108
109
q