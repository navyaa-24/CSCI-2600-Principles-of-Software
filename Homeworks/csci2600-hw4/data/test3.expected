First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from Troy Building to 87 Gymnasium:
	Walk East to (Ricketts Building)
	Walk East to (87 Gymnasium)
Total distance: 156.051 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from RPI Ambulance to Cogswell Laboratory:
	Walk North to (Intersection 99)
	Walk NorthWest to (Cogswell Laboratory)
Total distance: 143.488 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [Lally]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from Troy Building to 87 Gymnasium:
	Walk East to (Ricketts Building)
	Walk East to (87 Gymnasium)
Total distance: 156.051 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from Troy Building to 87 Gymnasium:
	Walk East to (Ricketts Building)
	Walk East to (87 Gymnasium)
Total distance: 156.051 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [bad1]
Unknown building: [bad2]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [bad2]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [bad1]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from Cogswell Laboratory to RPI Ambulance:
	Walk SouthEast to (Intersection 99)
	Walk South to (RPI Ambulance)
Total distance: 143.488 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from Cogswell Laboratory to MRC:
	Walk West to (Empire State Hall)
	Walk West to (MRC)
Total distance: 120.449 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from Rensselaer Apartment Housing Project RAHP A Site to VCC:
	Walk NorthWest to (Intersection 126)
	Walk SouthWest to (Burdett Avenue Residence Hall)
	Walk West to (Intersection 109)
	Walk SouthWest to (Intersection 118)
	Walk SouthWest to (Intersection 119)
	Walk NorthWest to (Admissions)
	Walk West to (Intersection 105)
	Walk West to (Intersection 108)
	Walk West to (Quadrangle Complex)
	Walk West to (87 Gymnasium)
	Walk West to (Ricketts Building)
	Walk West to (Troy Building)
	Walk SouthWest to (Intersection 133)
	Walk SouthWest to (Intersection 134)
	Walk SouthWest to (Lally Hall)
	Walk South to (Folsom Library)
	Walk East to (VCC)
Total distance: 1509.347 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from Folsom Library to Rensselaer Apartment Housing Project RAHP A Site:
	Walk North to (Lally Hall)
	Walk NorthEast to (Intersection 134)
	Walk NorthEast to (Intersection 133)
	Walk NorthEast to (Troy Building)
	Walk East to (Ricketts Building)
	Walk East to (87 Gymnasium)
	Walk East to (Quadrangle Complex)
	Walk East to (Intersection 108)
	Walk East to (Intersection 105)
	Walk East to (Admissions)
	Walk SouthEast to (Intersection 119)
	Walk NorthEast to (Intersection 118)
	Walk NorthEast to (Intersection 109)
	Walk East to (Burdett Avenue Residence Hall)
	Walk NorthEast to (Intersection 126)
	Walk SouthEast to (Rensselaer Apartment Housing Project RAHP A Site)
Total distance: 1430.347 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [108]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [108]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [108]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [108]
Unknown building: [109]
